### ISSUE TITLE
screenreader  mode does not work 

### ISSUE THREAD

**Issue by snufas** [Microsoft/vscode#68317]
Issue Type: <b>Bug</b>

hi, 
i am running latest NVDA screenreader on the latest version of vs code. the screen mode does not work. vs code detects the screenreader running and tells me to press alt + f1 to enable the mode. when i press the command it opens the menu window  instead. 

VS Code version: Code 1.31.0 (7c66f58312b48ed8ca4e387ebd9ffe9605332caa, 2019-02-05T22:35:56.624Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-5930K CPU @ 3.50GHz (12 x 3498)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.91GB (16.94GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|67%|

</details>Extensions: none

**Comment by snufas** [comment]
also one thing i noticed that after the vscode is clozed for a while  and opened again screenreader does not reed anything at all.

**Comment by snufas** [comment]
found the  saime issue on github 
https://github.com/Microsoft/vscode/issues/28316

**Comment by alexandrudima** [comment]
Let's track in #28316
